packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -315,8 +246,109 @@ import { name, version } from '../package'
 
   const startTime = Date.now()
 
+  // Engine check Listr. Separate Listr to avoid https://github.com/cenk1cenk2/listr2/issues/296
+  // Boolean flag
+  let hasPassedEngineCheck = null
+  // Array of strings
+  let engineErrorLog = []
+  // Docs link for engine errors
+  const engineErrorDocsLink = terminalLink(
+    'Tutorial - Prerequisites',
+    'https://redwoodjs.com/docs/tutorial/chapter1/prerequisites'
+  )
+
+  await new Listr(
+    [
+      {
+        title: 'Checking node and yarn compatibility',
+        skip: () => {
+          if (yarnInstall === false) {
+            return 'Warning: skipping check on request'
+          }
+        },
+        task: () => {
+          return new Promise((resolve) => {
+            const { engines } = require(path.join(templateDir, 'package.json'))
+
+            // this checks all engine requirements, including Node.js and Yarn
+            checkNodeVersion(engines, (_error, result) => {
+              if (result.isSatisfied) {
+                hasPassedEngineCheck = true
+                return resolve()
+              }
+              const logStatements = Object.keys(result.versions)
+                .filter((name) => !result.versions[name].isSatisfied)
+                .map((name) => {
+                  const { version, wanted } = result.versions[name]
+                  return style.error(
+                    `${name} ${wanted} required, but you have ${version}`
+                  )
```

Comment:
Let's decouple the styling from here so we can style it later

```suggestion
                  return `${name} ${wanted} required, but you have ${version}`
```